ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -95,6 +105,17 @@ export const NetworkListMenu = ({ onClose }) => {
 
   const orderedNetworksList = useSelector(getOrderedNetworksList);
 
+  const networkConfigurationChainIds = Object.values(networkConfigurations).map(
+    (net) => net.chainId,
+  );
+
+  const sortedFeaturedNetworks = FEATURED_RPCS.sort((a, b) =>
+    a.nickname > b.nickname ? 1 : -1,
+  ).slice(0, FEATURED_RPCS.length);
+
+  const notExistingNetworkConfigurations = sortedFeaturedNetworks.filter(
+    ({ chainId }) => !networkConfigurationChainIds.includes(chainId),
+  );
```

Comment:
maybe stuff that could be memoized with `useMemo` to avoid recalculating when unrelated stuff changes.  For example, dragging networks to a different order will re-render this  component, but shouldn't change this calculation.